modules/payloads/singles/windows/x64/pingback_reverse_tcp.rb

```diff
@@ -27,7 +31,18 @@ def initialize(info = {})
       'Handler'       => Msf::Handler::ReverseTcp,
       'Session'       => Msf::Sessions::Pingback
     ))
-    def generate_stage
+
+    def required_space
+      # Start with our cached default generated size
+      space = cached_size
+
+      # EXITFUNK 'seh' is the worst case, that adds 15 bytes
+      space += 15
+
+      space
+    end
+
+    def generate
```

Comment:
Was this intentional?

Reply:
Yes; now it matches the other singles in its area.

Reply:
Great!

Reply:
I think you are adjusting the other singles as we speak.